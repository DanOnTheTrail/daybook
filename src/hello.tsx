import React from 'react';

function Hello() : JSX.Element {
  return (
    <h2>Hello from React!</h2>
    );
}

export { Hello };