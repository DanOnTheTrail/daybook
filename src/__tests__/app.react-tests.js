import React from 'react';
import renderer from 'react-test-renderer';
import { Hello } from '../hello';

describe ("hello Component", () => {
  it('renders correctly', () => {
    expect.assertions(1);
    const tree = renderer
      .create(<Hello />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});