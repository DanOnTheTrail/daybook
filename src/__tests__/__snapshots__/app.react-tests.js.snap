// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hello Component renders correctly 1`] = `
<h2>
  Hello from React!
</h2>
`;
